### PR TITLE
Fix test dune file to not break dependants

### PR DESCRIPTION
**Pull request by ulrikstrid**

### CODE REVIEW

File: test/dune [hunk 1]
```diff
@@ -9,9 +8,8 @@
 
 (executable
  (name RunTests)
- (public_name RunTests.exe)
- (package brisk-reconciler)
- (libraries brisk-reconciler.test)
+ (libraries BriskReconcilerTest)
+ (preprocess (pps brisk_jsx ppx_deriving.std))
```
Comment (wokalski) [inline test/dune]: while we're on it, let's change the ppx name to brisk_ppx instead of brisk_jsx, shall we?

Comment (ulrikstrid) [reply test/dune]: Done

**Comment by ulrikstrid** [comment]
Running the tests this way puts the `junit.xml` somewhere that I can not find right now, will keep looking and update the PR.

**Comment by wokalski** [comment]
This is great, thank you.

**Pull request merged**